scripts/simplify.py

```diff
@@ -11,10 +11,30 @@
 	'style': 'display:inline'
 }
 
-default_trace_style = 'fill:none;stroke:#ff0000;stroke-width:1.25;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1'
-default_zone_style = 'opacity:0.5;fill:#00ffff;stroke:#000000;stroke-width:0.99999994px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1'
+default_trace_style = {
+	'fill': 'none', 
+	'stroke': '#ff0000', 
+	'stroke-width': '1.25', 
```

Comment:
No unit? `default_zone_style` uses pixels.

For compatibility with different files I'd prefer if we could use metric units, then we would have a fixed coordinate system and would only have to scale the images to match it.

Reply:
I've changed this to use '1.25px' so it's not left without a unit of measurement. b84721f642b9395b7a46015e009baf6e9d96db09

I would rather address units of measurements and everything associated with that in a separate issue.